src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -51,6 +51,10 @@ public class ConnectionService
         private const int DoesNotMeetPWReqs = 18466; // Password does not meet complexity requirements.
         private const int PWCannotBeUsed = 18463; // Password cannot be used at this time.
 
+        // Default SQL constants (required to ensure connections such as serverless are able to connect and retry properly).
+        private const string DefaultApplicationName = "azdata";

```

Comment:
We already have ApplicationName on line 177 that stores the default application name so this is not necessary.